data-migrations/database-init.sh

```diff
@@ -27,8 +27,18 @@ CREATE_DATABASE="CREATE DATABASE IF NOT EXISTS ${MYSQL_DATABASE}
   CHARACTER SET utf8mb4
   COLLATE utf8mb4_unicode_ci;"
 
-echo "-h${MYSQL_HOST} -P${MYSQL_PORT} -u${MYSQL_USER} -p ${MYSQL_PASSWORD}"
-
 # Create the Database
 echo "Creating database ${MYSQL_DATABASE} ..."
-mysql -h${MYSQL_HOST} -P${MYSQL_PORT} -u${MYSQL_USER} -p ${MYSQL_PASSWORD} <<< ${CREATE_DATABASE}
+mariadb -h${MYSQL_HOST} -P${MYSQL_PORT} -u${MYSQL_USER} -p${MYSQL_PASSWORD} <<< ${CREATE_DATABASE}
```

Comment:
@briri - Should this file be mentioned in the README.md file to initialize the database?